service/controller/resource/monitoring/scrapeconfigs/resource.go

```diff
@@ -201,7 +201,7 @@ func getTemplateData(ctx context.Context, ctrlClient client.Client, cluster meta
 
 func getDefaultAppVersion(ctx context.Context, ctrlClient client.Client, cluster metav1.Object, config Config) (string, error) {
 	appName := fmt.Sprintf("%s-default-apps", key.ClusterID(cluster))
-	appNamespace := fmt.Sprintf("org-%s", key.GetOrganization(cluster))
+	appNamespace := cluster.GetNamespace()
```

Comment:
If this is true, sure, but I would rather kaas teams validate the org label

Reply:
@giantswarm/area-kaas  Do we have an admission that validates the org label?

Reply:
If sometimes the label is wrong, then I don't think we do :sweat_smile: 

When it's wrong, what's its value? what does wrong mean here?

Reply:
Sometimes users created clusters with a wrong org label.
For example, they changed the namespace of the app.
But the cluster CR still points to org-giantswarm..

Reply:
If I remember correctly, we decided to remove the organization label during a KaaS sync last year. The reasoning being that the namespace is the source of truth for the organization. So I think it's good that you move away from using it.

Reply:
I really wonder if those decisions should not be shared in news channel or in adrs because this is a pain to track all those changes that are impacting for consumers 😅 Thanks @fiunchinho